ui/ui-community/pages/session/sign-in.tsx

```diff
@@ -7,16 +7,37 @@ export default function SignInPage(props) {
   return (
     <>
       <Head>
-        <title>Grouparoo: Sign In</title>
+        <title>Grouparoo</title>
       </Head>
 
-      <h1>Sign In</h1>
-
-      <Row className="border-between-columns">
-        <Col>
-          <SignIn {...props} useApi={useApi} />
-        </Col>
-      </Row>
+      <div style={{ width: "100%", height: "100%" }}>
+        <Row>
+          <Col style={{ textAlign: "center" }}>
+            <Image
+              style={{ maxHeight: 50, margin: 40 }}
+              src="/public/images/logo/logo-and-wordmark-black-words.svg"
+            />
+            <br />
+            <h2>Sync, Segment, and Send your Product Data Everywhere</h2>
+            <p>
+              Grouparoo is open source data infrastructure to keep customer data
+              in-sync between your data warehouse and 3rd party tools.
+            </p>
+            <p>
+              Learn more at{" "}
+              <a target="_blank" href="https://www.grouparoo.com">
+                www.grouparoo.com
+              </a>
+            </p>
+            <br />
+          </Col>
+        </Row>
```

Comment:
If the user will see the common (ui-components) index page, they will already have seen this information. Why show it again on the sign in page?

Reply:
@seancdavis explained it well below.  The way things redirect now, it automatically pushes them to /session/sign-in currently... so they don't see that initial landing page in config (as things are currently)

Reply:
Is this a concern now that you aren't using redirects?

Reply:
All 3 sign-in pages `ui-enterprise/pages/session/sign-in.tsx`, `ui-community/pages/session/sign-in.tsx` and `ui-config/pages/session/sign-in.tsx` have been modified in this PR to copy over the content from `/`.  Perhaps now that there aren't redirects, that's not the case?